src/api/matrix.js

```diff
@@ -0,0 +1,36 @@
+import Http from '../http';
+import Navigator from '../navigator';
+
+class Matrix {
+  static async getMatrixDistances(routingOptions={}) {
+    if (!routingOptions.origins) {
+      const { latitude, longitude } = await Navigator.getCurrentPosition();
+      routingOptions.origins = { latitude, longitude };
```

Comment:
I think maybe this should be an array? since the matrix API takes an array of coordinates

Reply:
@jsani-radar I think the Matrix API just takes a pipe-delimited string: https://radar.com/documentation/api#matrix

> origins (string, required): A list of origins. A pipe-delimited string in the format latitude0,longitude0|latitude1,longitude1|....

Reply:
oh, wait, I see what you're saying...let me look into that

Reply:
yup makes sense - though I'm not seeing where this is transformed to a pipe-delimited string, as it seems like we're passing the object arg through to the API

Reply:
what I mean here is that we need to transform the `origins` var to the format that the API expects, see this example for what we do in our `searchPlaces` call with chains / categories / groups - https://github.com/radarlabs/radar-sdk-js/blob/master/src/api/search.js#L22-L30

they're [passed in as arrays](https://github.com/radarlabs/radar-sdk-js/blob/master/test/api/search.test.js#L21-L23), but then formatted correctly

Reply:
yeah +1, in this spot we can set origins to be an array like:

```
const { latitude, longitude } = await Navigator.getCurrentPosition();
routingOptions.origins = [{ latitude, longitude }];
```